### PR TITLE
Refactor parent guide info page

### PR DESCRIPTION
**Pull request by sukritsangvong**
Change list:
- Add icon to ParentGuide info page
- Change text styling of the heading, subheading, and the card
  - The card now consists of `heading` `subheading` and `content` where content can be as many strings as possible

Note: the size of the text is a bit bigger than the design cause it looks way too small on the big screen. The design's font size is 18px. I made it 20. The icon size on the design was 29x29. I made it 35x35

image
image
image

### CODE REVIEW

File: pages/ParentGuide.js [hunk 1]
```diff
@@ -400,22 +400,22 @@ export const ParentGuideInformation = ({ route, navigation }) => {
             >
                 <InformationContainer>
                     <StandardTextbox
```
Comment (RichartE) [inline pages/ParentGuide.js:402]: You may want to reduce the weight of the category title to match Figma.

Comment (RichartE) [reply pages/ParentGuide.js:402]: This is a design thing, but I feel that INTERPRETATIONS followed by People, Place etc. doesn't make sense to me. I would just have the interpretation (The people your child mention they find trustworthy or whatever). What do others think?

**Review by RichartE — APPROVED** [review]
I left some minor comments but it looks good.

**Comment by RichartE** [comment]
This is a design thing, but I feel that looking at the pictures above, INTERPRETATIONS followed by People, Place etc. doesn't make sense to me. I would just have the interpretation directly following (The people your child mention they find trustworthy or whatever). What do others think?

File: pages/ParentGuide.js (hunk 1 above)
Comment (sukritsangvong) [reply pages/ParentGuide.js:402]: I decided to bump the weight up a bit or else everything will get too light.

Comment (sukritsangvong) [reply pages/ParentGuide.js:402]: (It looks way lighter than figma's)

**Pull request merged**